File: js/1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const numero = parseInt(lines.shift());
const horasTrabalhadas = parseFloat(lines.shift());
const valorHora = parseFloat(lines.shift());
const salario = horasTrabalhadas * valorHora;
console.log(`NUMBER = ${numero}\nSALARY = U$ ${salario.toFixed(2)}`);

File: cs/1008.cs
int numero = int.Parse(Console.ReadLine());
double horasTrabalhadas = double.Parse(Console.ReadLine());
double valorHora = double.Parse(Console.ReadLine());
double salario = horasTrabalhadas*valorHora;
Console.WriteLine($"NUMBER = {numero}\nSALARY = U$ {salario.ToString("0.00")}");